crates/node/src/main.rs

```diff
@@ -140,22 +153,20 @@ impl workflow::TransactionStore for storage::Database {
     }
 }
 
-struct AuthenticatingTxHandler {
```

Comment:
Authorizing in `mempool` is better approach than in P2P handler. `mempool` captures both, JSON-RPC & P2P.